### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by arshasherin**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -78,3 +78,4 @@
   }
 }
 whoooos
+impooort
```
Comment (arshasherin) [inline index.html]: please remove sherinshaaz import

Comment (arshasherin) [reply index.html]: nwmdbmw,dwd

Comment (arshasherin) [inline index.html]: remove

**Review by sherinshaaz — CHANGES_REQUESTED** [review]
dfsdfsdf

File: index.html [hunk 2]
```diff
@@ -78,3 +78,4 @@
   }
 }
 whoooos
```
Comment (arshasherin) [inline index.html]: remove whoos

**Review by arshasherin — COMMENTED** [review]
esdfsdf

File: index.html [hunk 3]
```diff
@@ -77,4 +77,5 @@
     );
   }
 }
-whoooos
+
+impooort
```
Comment (arshasherin) [inline index.html]: remove import

**Pull request merged**